README.md

```diff
@@ -30,9 +30,9 @@ Once these types have been defined, you can instatiate them and access their dat
 ```c++
 //create the DynamicData accordingly with the recently created "Outer" DynamicType
 DynamicData data(outer);
+
 //write value
-data["b"]["a"].value(42);
```

Comment:
So we decided to remove the functionality .value to do assignation? or we are just removing it from the documentation?

Reply:
Only from this part of the documentation (the first lines), to keep it simpler only show at this moment the *assignment* way.

Reply:
it makes sense. agreed